internal/tweets/creatTweet.go

```diff
@@ -0,0 +1,45 @@
+package tweets
+
+import (
+	"Twitter_like_application/internal/database/pg"
+	"Twitter_like_application/internal/services"
+	"encoding/json"
+	"github.com/go-playground/validator/v10"
+	"net/http"
+	"time"
+)
+
+func CreateNewTweet(w http.ResponseWriter, r *http.Request) {
+	tweetValid := &TweetValid{
+		Validate: validator.New(),
+		ValidErr: make(map[string]string),
+	}
+	if err := RegisterUsersValidations(tweetValid); err != nil {
```

Comment:
does it really have to do anything with "Users"?

Reply:
done